[SYCL][NFC]  Remove `sampled_image_accessor` and `unsampled_image_accessor` from test plan for free function kernels extension (#19086)
intel/llvm: sycl/test-e2e/FreeFunctionKernels/test-plan.md

A series of checks should be performed that we can pass `local_accessor` 
as kernel parameter to free function kernel and use it within kernel.

> #### Test structs that contain one of the following `accessor`, `local_accessor`, types when used as kernel parameter:

Reviewer: ```suggestion
#### Test structs that contain either `accessor` or `local_accessor` when used as kernel parameter:
```

A series of checks should be performed that we can pass struct that contain 

> one of the following `accessor`, `local_accessor` or types as kernel parameter 

Reviewer: ```suggestion
either `accessor` or `local_accessor` as kernel parameters 
```
